dino.icu.yaml

```diff
@@ -172,6 +172,12 @@ geta: # yoda - https://getadinoicu.yodacode.repl.co/dino.png - get a dino from h
   type: CNAME
   value: ef35bb48-f121-4f7f-91f6-93f601559c24.id.repl.co.
 
+heramb:
+  ttl: 600
+  type: A
+  value: 76.76.21.21
+  
```

Comment:
```suggestion
heramb:
  ttl: 600
  type: CNAME
  value: cname.vercel-dns.com.
```

Reply:
did the suggested changes 👍  @alx-alexpark